csp-guru/datapackage.json

```diff
@@ -92,128 +92,98 @@
                         "type": "number"
                     },
                     {
-                        "name": "Total_cost_[million_currency;non-deflated]",
-                        "description": "construction cost in million local currency, non-deflated",
-                        "type": "number"
```

Comment:
It's a pity that you changed the type of all these columns to the very generic "string" type. If you could change that easily I'd do it, but it's not a show stopper.

Reply:
number is a string type?

Reply:
Nope. Number means numerical and is more restrictive than string. String is everything from "256.45" to "256.45 €/MW" to "richard is a pretty nice guy". So if the type was number, only the former would be valid. As a string, each of those is valid. To process the data, a more restrictive and more formal type is better. But again, this shouldn't be a show stopper.

Reply:
ich versteh nicht wie das sonst gehen soll. cent beträge: 0.23 ist "number", oder?

Reply:
Ja, das Problem ist das Symbol der Währung. Und eigentlich ist das auch überflüssig, weil die Währung ja in jeweils einer anderen Spalte angegeben wird. Also zum Beispiel: "Support_currency" gibt die Währung der "Remuneration_[currency/kWh]" oder? Falls ja, ist es nicht nötig, dass die Remuneration mit "€ 0.23 " angegeben wird. Stattdessen sollte es "0.23" sein, und dann kann der Typ auch "Number" sein.

Reply:
Bei "Remuneration_[USD2018;deflated]" und "LCOE_5_25_[USD2018/kWh]" ist die Situation noch klarer: da ist ein Dollarsymbol komplett überflüssig.

Reply:
Also, ich würde bei den drei Spalten das Einheitssymbol aus den Daten nehmen und dann den Typ auf "number" stellen.

* Remuneration_[USD2018;deflated]
* LCOE_5_25_[USD2018/kWh]
* Remuneration_[currency/kWh]

Reply:
aber das war schon number?